docs/02.API-REFERENCE.md

```diff
@@ -39,14 +39,20 @@ Enum that contains JerryScript **object** value types:
 
  - JERRY_OBJECT_TYPE_NONE - Non-object type
  - JERRY_OBJECT_TYPE_GENERIC - Generic JavaScript object without any internal property
+ - JERRY_OBJECT_TYPE_NAMESPACE - Namespace object
  - JERRY_OBJECT_TYPE_ARRAY - Array object
  - JERRY_OBJECT_TYPE_PROXY - Proxy object
  - JERRY_OBJECT_TYPE_SCRIPT - Script object (see [jerry_parse](#jerry_parse))
  - JERRY_OBJECT_TYPE_MODULE - Module object (see [jerry_parse](#jerry_parse))
+ - JERRY_OBJECT_TYPE_PROMISE - Promise object
+ - JERRY_OBJECT_TYPE_DATAVIEW - Dataview object
  - JERRY_OBJECT_TYPE_FUNCTION - Function object (see [jerry_function_get_type](#jerry_function_get_type))
  - JERRY_OBJECT_TYPE_TYPEDARRAY - %TypedArray% object (see [jerry_get_typedarray_type](#jerry_get_typedarray_type))
  - JERRY_OBJECT_TYPE_ITERATOR - Iterator object (see [jerry_iterator_get_type](#jerry_get_typedarray_type))
+ - JERRY_OBJECT_TYPE_SITERATOR - String iterator object
```

Comment:
SITERATOR :)
Iterator is iterator, `jerry_iterator_get_type` should provide more detailed info.